ScipDotnet.Tests/SnapshotTests.cs

```diff
@@ -130,12 +130,7 @@ private static string DiffStrings(string obtained, string expected)
     private static string[] ListSnapshotInputDirectories()
     {
         var inputs = Path.Join(RootDirectory(), "snapshots", "input");
-
-#if NET7_0
-        return Directory.GetDirectories(inputs).Where(x => !x.Contains("WpfApplication")).ToArray();
```

Comment:
Remove this since it is no longer needed with the addition of the `<EnableWindowsTargeting>` attribute